tests/test_models_candles.py

```diff
@@ -148,9 +148,9 @@ def test_candlestick_periods(interval, expected1, expected2):
 def test_candlestick_timezone():
     known = pendulum.datetime(2022, 1, 1, 0, 0)
     with pendulum.test(known):
-        cdl = CandleStick(symbol="NIFTY", timezone="Asia/Kolkata")
-        assert cdl.timer.start_time.timezone_name == "Asia/Kolkata"
-        assert cdl.periods[0].timezone_name == "Asia/Kolkata"
+        cdl = CandleStick(symbol="NIFTY", timezone="local")
```

Comment:
We are checking a specific timezone here and the `CandleStick` class defaults to `local` timezone.
I assume you are testing whether the local timezone and the timer timezone are same if provided the `local` argument. Is this the case?

Reply:
I am testing in HK/SG timezone - the intent of test case is to check of the timezone name and value propagated correctly - the modification check that correctly and independently leveraging the name form now api of pendulum. 

When you test in Indian TZ - you will get it right Asia/Kolkata - when I am testing this in HK timezone the test case is failing - this change helps fix that while retaining the validation. Longer term - there is need to refactor the test cases and bring TZ specific tests that pass irrespective of TZ, the package user runs the tests in

Reply:
Got it